code/modules/surgery/dental_implant.dm

```diff
@@ -20,7 +20,7 @@
 	tool.loc = target
 
 	var/datum/action/item_action/hands_free/activate_pill/P = new
-	P.button_icon_state = tool.icon_state
```

Comment:
It works in my tests. I think the problem is that item actions use the icon of the target item by default and that line tried to find nonexistent pill icon states in actions.dmi.